packit_service/service/urls.py

```diff
@@ -20,49 +20,47 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from flask import url_for
+from packit_service.config import ServiceConfig
 
-from packit_service.service.app import packit_as_a_service as application
+DASHBOARD_URL = ServiceConfig().get_service_config().dashboard_url
 
 
-def get_srpm_log_url_from_flask(
+def _get_dashboard_url(build: str, id_: int) -> str:
```

Comment:
What about something like `_get_dashboard_result_url` or something like that?

(I expect DASHBOARD_URL as a return value of a function with this name..;)